concepts/05 Widgets/Calendar/05 Handle the Value Change Event.md

```diff
@@ -39,6 +37,73 @@ To process a new calendar value, you need to handle the value change event. If t
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxCalendar
+            v-model="calendarValue"
+            @value-changed="calendarValueChanged"
+        />
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import DxCalendar from 'devextreme-vue/calendar';
+
+    export default {
+        components: {
+            DxCalendar
+        },
+        data() {
+            calendarValue: new Date()
+        },
+        methods: {
+            calendarValueChanged: function({ previousValue, value: newValue }) {
```

Comment:
I know that destructuring is all the rage, but these are tutorial examples. Let's keep them as clear as possible, without shortcuts that not everyone know.

Reply:
got it